lua/core/menu/home.lua

```diff
@@ -0,0 +1,82 @@
+local m = {}
+
+m.pos = 1
+m.list = {"SELECT", "SYSTEM", "SLEEP"}
+
+m.init = function()
+  menu.timer.time = 1
+  menu.timer.count = -1
+  menu.timer.event = function() menu.redraw() end
+  menu.timer:start()
+end
+
+m.deinit = function()
+  menu.timer:stop()
+end
+
+m.key = function(n,z)
+  if n == 2 and z == 1 then
+    menu.showstats = not menu.showstats
+    menu.redraw()
+  elseif n == 3 and z == 1 then
+    menu.set_page(m.list[m.pos])
+  end
+end
+
+m.enc = function(n,delta)
+  if n == 2 then
+    m.pos = util.clamp(m.pos + delta, 1, #m.list)
+    menu.redraw()
+  end
+end
+
+m.redraw = function()
+  screen.clear()
+  menu.draw_panel()
+  for i=1,3 do
+    screen.move(0,25+10*i)
+    if(i==m.pos) then
+      screen.level(15)
+    else
+      screen.level(4)
+    end
+    screen.text(m.list[i].." >")
+  end
+
+  if not menu.showstats then
+    screen.move(0,15)
+    screen.level(15)
+    local line = string.upper(norns.state.name)
+    if(menu.scripterror and menu.errormsg ~= 'NO SCRIPT') then
```

Comment:
=>
```lua
  if (menu.scripterror and menu.errormsg ~= 'NO SCRIPT') then
```

(space after `if`)

aside: maybe we should consider an auto-formatter?